Shoe.h

```diff
@@ -0,0 +1,24 @@
+struct Shoe
```

Comment:
Where is the `#pragma once`?
Where are the `#include` statements for the non-primitive types and JUCE macro?
- [ ] revise